.gitignore

```diff
@@ -68,3 +68,6 @@ snapshots.js
 
 # Yarn local mirror content
 .yarn-local-mirror
+
+# Ignore the generated antlr files
+/src/plugins/data/public/antlr/opensearch_sql/grammar/.antlr
```

Comment:
i see we are git ignoring this.

do we have a script that ends up generating the rules? or how do we get these values now?

Reply:
This is neither ignoring grammar files or generated lexer and parser files, this is for ignoring some intermediate local, auto generated files that are not needed for autocomplete logics or file reference, the entire .antlr folder is generated when you have changes to your grammar for development work. The grammar is located at `/src/plugins/data/public/antlr/opensearch_sql/grammar/` and generated lexer and parser are located at `/src/plugins/data/public/antlr/opensearch_sql/.generated`, the command to generate lexer and parser are located in package.json and they are already generated that code has reference to.